admin/assets/stylesheets/elements/_pupils.scss

```diff
@@ -33,3 +33,32 @@
     width: 60%;
   }
 }
+
+/* Add/edit form */
+
+.upn-text {
```

Comment:
Please revert the style change associated with the changes to the UPN text on add/edit pupil.  This is covered under a different ticket using `<detail><summary>` tags.